examples/gno.land/r/gov/dao/v2/render.gno

```diff
@@ -0,0 +1,124 @@
+package govdao
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/dao"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/moul/txlink"
+	"gno.land/r/demo/users"
+)
+
+func Render(path string) string {
+	var out bytes.Buffer
```

Comment:
It seems you want to use `strings.StringBuilder` instead of `bytes.Buffer`.

Reply:
I reverted to just a simple string for readability.